src/Components/Steps/SelectStatePage.tsx

```diff
@@ -12,7 +12,7 @@ import { useUpdateWhiteLabelAndNavigate } from '../RouterUtil/RedirectToWhiteLab
 import QuestionDescription from '../QuestionComponents/QuestionDescription';
 import { zodResolver } from '@hookform/resolvers/zod';
 
-export const STATES: { [key: string]: string } = { co: 'Colorado', nc: 'North Carolina' };
+export const STATES: { [key: string]: string } = { co: 'Colorado', nc: 'North Carolina', il: 'Illinois' };
```

Comment:
We should wait to do this until IL is ready to be used by everyone. It adds it to the state dropdown if you don't have a state selected. https://screener.myfriendben.org/select-state
```suggestion
export const STATES: { [key: string]: string } = { co: 'Colorado', nc: 'North Carolina' };
```